src/app/store/slices/storage/storage.thunks/createFolderThunk.ts

```diff
@@ -106,7 +106,9 @@ export const createFolderThunkExtraReducers = (builder: ActionReducerMapBuilder<
       if (requestOptions?.showErrors) {
         const errorMessage = action.error.message?.includes('already exists')
           ? t('error.folderAlreadyExists')
-          : t('error.creatingFolder');
+          : action.error.message?.includes('Invalid folder name')
```

Comment:
Same here

Reply:
Here is the same, we always get a 400, so we cannot separe messages.